### PR TITLE
FRAN-140: Add Docker file to build ubuntu image for Staging stop-and-fix

### PR DESCRIPTION
**Pull request by kcguttikonda**
1. Added Docker File
2. Configured the project in cloudelements docker hub to automatically build docker image whenever we change docker file

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,47 @@
+FROM  ubuntu:18.04
+MAINTAINER developer@cloud-elements.com
+ENV USER=root
+ENV CHURROS_TEMPLATE="/qaAutomation/sauce.json"
+ENV CHURROS_USER="claude.elements.qa+circleci@gmail.com"
+ENV CHURROS_URL="snapshot.cloud-elements.com"
```
Comment (patrickdechant) [inline Dockerfile]: shouldn't this be the staging url

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ENV CHURROS_PASSWORD="Cl0ud3l3m3nts!"
+ENV CHURROS_ENV="snapshot"
```
Comment (patrickdechant) [inline Dockerfile]: staging?

File: Dockerfile (hunk 1 above)
Comment (kcguttikonda) [reply Dockerfile]: oops. Updated url, environment  and creds.

File: Dockerfile (hunk 2 above)
Comment (kcguttikonda) [reply Dockerfile]: oops. Updated url, environment  and creds.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,47 @@
+FROM  ubuntu:18.04
+MAINTAINER developer@cloud-elements.com
+ENV USER=root
+ENV CHURROS_TEMPLATE="/qaAutomation/sauce.json"
```
Comment (patrickdechant) [inline Dockerfile]: One more question; does `CHURROS_TEMPLATE="/qaAutomation/sauce.json"` need to be in here?

Comment (kcguttikonda) [reply Dockerfile]: Can be removed. Removed it.

**Pull request merged**